llarp/config/config.hpp

```diff
@@ -72,11 +72,11 @@ namespace llarp
     std::string m_ifname;
     std::string m_ifaddr;
 
-    std::string m_keyfile;
+    std::optional<fs::path> m_keyfile;
     std::string m_endpointType;
     bool m_reachable = false;
-    int m_hops = -1;
-    int m_paths = -1;
+    std::optional<int> m_Hops;
+    std::optional<int> m_Paths;
     bool m_AllowExit = false;
```

Comment:
Did this fix something, or is it more of a clean up thing?

Reply:
it's a cleanup